src/main/java/org/jabref/model/util/OptionalUtil.java

```diff
@@ -41,4 +43,16 @@
     public static <T, R> Stream<R> flatMap(Optional<T> value, Function<? super T, ? extends Collection<? extends R>> mapper) {
         return toStream(value).flatMap(element -> mapper.apply(element).stream());
     }
+
+    public static <T> Boolean isPresentAnd(Optional<T> value, Predicate<T> check) {
```

Comment:
I also don't really like them but still found them more readable than the inlined code. I will keep them for the moment but feel free to remove them if you really dislike.